libs/core/src/lib/quick-view/quick-view-group/quick-view-group.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { QuickViewGroupComponent } from './quick-view-group.component';
+
+describe('QuickViewGroupComponent', () => {
+    let component: QuickViewGroupComponent;
+    let fixture: ComponentFixture<QuickViewGroupComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [QuickViewGroupComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(QuickViewGroupComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
```

Comment:
Is there any reason why we don't have any specs not having test scripts?

Reply:
@sKudum Could you please guide me, which tests are you expected here? 
No one of the components has no methods or "dynamic behavior", so unit tests are not applicable for it as I see it

Reply:
Was wondering, testing a dummy component with template ready will help to test the component. 
such as checking whether expected tag is present and it has the value or id or class name present, is it supporting accessibility  properties. etc

Reply:
@sKudum I've added tests in those places where items have binding/directive/attributes which made a sort of "dynamic behavior" 🙂